packages/replay/src/coreHandlers/util/fetchUtils.ts

```diff
@@ -78,33 +80,37 @@ async function _prepareFetchData(
   const {
     url,
     method,
-    status_code: statusCode,
+    status_code: statusCode = 0,
```

Comment:
Should this default to `0`? Does `undefined` vs explicit `0` have different meanings?

Reply:
We've been doing this before as well (only setting it further down) - but 🤷 I'd say it's fine?